metabase/metabase
# fix(sdk): entity picker's hover and accent colors are not mappable via sdk theming

Closes https://github.com/metabase/metabase/issues/47560

## Known Issues

The right chevron icon on the collection picker, as well as the "Search Everywhere / Our Analytics" selector is not yet themed. I'll work on these either before we land the PR, or on a follow-up PR that polishes these issues.

![CleanShot 2567-11-21 at 03 02 06@2x](https://github.com/user-attachments/assets/cf237ed5-bbb4-4520-add4-53375615211c)

![CleanShot 2567-11-21 at 03 02 18@2x](https://github.com/user-attachments/assets/87e7be0f-bbd2-42e1-9ff4-e79351c2a829)

## Description

Adds the necessary hover

### How to verify

- Open Storybook
- Go to "InteractiveQuestion > Themed > Dark Theme Editor" story
- Play around with it
- Hover colors should be well-themed on dark mode now

### Demo

![CleanShot 2567-11-21 at 03 04 26@2x](https://github.com/user-attachments/assets/142b4e7d-1463-4dab-8b9c-f3134618b9d6)

![CleanShot 2567-11-21 at 03 04 31@2x](https://github.com/user-attachments/assets/fa9e9f7e-22c4-41bb-9db9-f5855aa8acf6)

### Checklist

- [ ] Tests have been added/updated to cover changes in this PR - to add either visual tests or unit tests
